server/src/lib/score-import/import-types/file/wacca-mypage-scraper/types.ts

```diff
@@ -0,0 +1,31 @@
+export interface MyPageRecordsRawCsvRecord {
+
```

Comment:
please tell me how to make this pass lint

Reply:
it's a cadence bug. ignore it with // eslint-ignore